### ISSUE TITLE
Manual update and more:

### ISSUE THREAD

**Issue by Hakansv** [Rasbats/opencpn-plugins-manual#1]
Mike..
We would continue here instead of the closed PR on radar_pi?
Text from your last mes: 

> _(You are live on the manual website!
> https://rasbats.github.io/opencpn-plugins-manual/opencpn-plugins-manual/0.1/index.html
> Top right of your pages you will see 'Edit this page'. Using this allows edits of the manual on Github. On the next build of your plugin the edits will be seen in the manual (after the main manual is updated).)"_

Yes, I see - fine. I've added some small edits for test. Beyond it a link to the source at Github. But it's not working as it should in Firefox. Could you update this site again for a check?

> _BTW if you need HTML for a help button you will find it at:
> https://github.com/Rasbats/opencpn-plugins-manual/tree/main/docs/radar/0.1
> This could be placed in a folder in your data directory and referenced by the plugin._

Please elaborate I don't understand this feature.

New:
1. If we release a new version of radar_pi how would you catch that to update this page?
2. What's in plan for the future? I suppose you would not keep this site alive for ever? (Although really appreciated so far)
3. I've added a text in "home.adoc" to detail the plugin version covered by the manual. Could we instead use the manual version stated in antora.yml for the same purpose?

Thanks
Håkan

**Comment by Rasbats** [comment]
Håkan...
Fine to discuss here.

| **Could you update this site again for a check?**

I assume you have commited the changes, to update your radar_pi repo. The next time the build of the manual (OPM) takes place it will pull in those changes on my clone. Then a push to the repo for the manual and you will see the changes. It will only be visible in Firefox after that. There is a way to add stuff to the manual folder to make the html locally but this involves installing antora. This could become part of the wiki/manual eventually. Not important for the functioning of the plugins manual.

Can you confirm that you commited the changes?  
At the top of the commit message add [skip ci][skip appveyor] if you want to stop a build of the plugin. Because there will be no tag this only ends up in opencpn-alpha, nowhere else.

| **This could be placed in a folder in your data directory and referenced by the plugin.**
Have a look here:
[https://github.com/Rasbats/otidalplan_pi/data](https://github.com/Rasbats/otidalplan_pi/data)

"TidalPlanInformation.html" has the plugin guide (This plugin is beta).

Referenced by the function :OnInformation(wxCommandEvent& event) in:
[https://github.com/Rasbats/otidalplan_pi/blob/master/src/otidalplanUIDialog.cpp](https://github.com/Rasbats/otidalplan_pi/blob/master/src/otidalplanUIDialog.cpp)  

1.   If we release a new version of radar_pi how would you catch that to update this page?

Good point: the maintainer of the OPM would need to know. If not any change to the manual would only happen if the OPM         was updated for any reason.

2.     What's in plan for the future? I suppose you would not keep this site alive for ever? (Although really appreciated so far)

 I want the repo for the manual to be hosted elsewhere (with access) as soon as possible. We will be in discussion with Dave about this shortly. My preference would be https://opencpn/opencpn-plugins-manual. The dev and user manuals need a similar home.

3.     I've added a text in "home.adoc" to detail the plugin version covered by the manual. Could we instead use the manual version stated in antora.yml for the same purpose?

Good idea: I will propose that to the team. The OPM is changing constantly. The version number is required by antora but it does not have to be '0.1'.

Thanks for the interesting points.

**Comment by leamas** [comment]
> If we release a new version of radar_pi how would you catch that to update this page?

There should be a 'latest' version available to use. There are hooks in Antora for this, might need to be explored.  Since github pages seemingly runs on some sane host, it should also possible to use a simple symlink. But either way, external code should IMHO not have to worry about actual version used unless they really want to.

EDIT: Also note that a manual available on this startpage: `https://leamas.github.io/ocpn-manuals/ocpn-dev-manual/5.3.1/intro-AboutThisManual.html ` also is available at  `https://leamas.github.io/ocpn-manuals/ocpn-dev-manual/`; the latter will use the default version as defined in antora.yml

**Comment by Rasbats** [comment]
But we are talking about the individual plugin module, managed by the developer?

**Comment by Hakansv** [comment]
Mike..
Since I'm one of the owners my edits are merged at once, so yes https://github.com/opencpn-radar-pi/radar_pi  is updated.
I didn't thought of the "[skip ci][skip appveyor] " I saw you was using so all was built. I do hope with any further harm.
BTW: you PR was building the appveyor but not ci so the [skip appveyor]  part may not worked?

Manual update at plugin release.... 
If we decide to use the version number in antora.yml could a change to that version be a trig?

"TidalPlanInformation.html"..
I'll have look at otidalplan to learn how you used that.

Thanks
Håkan

**Comment by leamas** [comment]
> But we are talking about the individual plugin module, managed by the developer?

yes. But  there should IMHO still be a possibility to use the "latest" or "default" version from outside, somehow. It should be possible to fix.

**Comment by Hakansv** [comment]
Mike..
Now I understand my Links error in last commit. (embarrassed!) Now corrected on radar_pi GitHub
Håkan

**Comment by Rasbats** [comment]
Håkan... No worries. Lots of learning taking place here ;)
I will do some research to stop builds.

**Comment by Rasbats** [comment]
You should see your latest edits. Manual has been updated.

**Comment by Rasbats** [comment]
About the [skip CI].

Some issues raised about this. Apparently implemented in github actions.
Alec... do you know how to implement this?

**Comment by leamas** [comment]
I don't think we need to implement anything. [skip ci] is directly supported by both circleci and drone.io (just read about it, not tested) No idea about appveyor.

**Comment by leamas** [comment]
Appveyor seems to support [skip ci] after some configuration: https://www.appveyor.com/docs/how-to/filtering-commits/

**Comment by Hakansv** [comment]
Mike..
Since we have a released a master version of radar_pi it would be nice to refer to the new manual here.
Would it be feasibly to link to "your" version on the forum until a standard solution? Or do you prefer we wait?
Håkan

**Comment by leamas** [comment]
Please wait. The new manual is still a WIP

**Comment by Hakansv** [comment]
OK No problem
